Package.swift

```diff
@@ -616,13 +616,10 @@ let package = Package(
       dependencies: ["FirebaseDatabase", "FirebaseDatabaseSwift"],
       path: "FirebaseDatabaseSwift/Tests/"
     ),
-    .target(
+    .binaryTarget(
       name: "FirebaseSharedSwift",
-      path: "FirebaseSharedSwift/Sources",
-      exclude: [
-        "third_party/FirebaseDataEncoder/LICENSE",
-        "third_party/FirebaseDataEncoder/METADATA",
-      ]
+      url: "https://dl.google.com/firebase/ios/experimental/bin/sharedswift/10.16.0/FirebaseSharedSwift.zip",
+      checksum: "cfa32eb16ba8fafa7a443ab8f48e019794470bb358066d6c571df1531df02fe6"
```

Comment:
Is there a corresponding release process update for this?

Reply:
Yes, there will need to be. I'm thinking we can extend the tooling used to handle the Firestore binary to also handle SharedSwift.